reference/docs-conceptual/install/Installing-PowerShell-Core-on-macOS.md

```diff
@@ -6,8 +6,9 @@ ms.date: 08/20/2020
 
 # Installing PowerShell on macOS
 
-PowerShell supports macOS 10.13 and higher. All packages are available on our GitHub [releases][]
-page. After the package is installed, run `pwsh` from a terminal.
+PowerShell supports macOS 10.12 and higher. PowerShell 7.0.3 and PowerShell Preview 7.1 require
+macOS 1.3 and higher. All packages are available on our GitHub [releases][] page. After the package
```

Comment:
Shouldn't that be "macOS 10.13" and not "1.3"? :)

Reply:
Yes, sorry this was a rushed push to change branches. I'm changing this to WIP as I'm waiting for a response.